### PR TITLE
fix: Correct JFrog X-Ray audit issues

### PR DESCRIPTION
**Pull request by erik-catalao**
# Description

Updates dependencies in order to no longer have any JFrog X-Ray audit issues

Fixes # (issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [ ] I have added tests to cover my changes
-   [ ] I have made corresponding changes to the documentation

### Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)

### CODE REVIEW

**Review by ruiqbarbosa — APPROVED** [review]

File: tests/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj [hunk 1]
```diff
@@ -15,12 +15,14 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="AutoFixture" Version="4.17.0" />
+        <PackageReference Include="AutoFixture" Version="4.18.1" />
+		    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```
Comment (joelfoliveira) [inline tests/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj]: please fix indentation.

File: tests/KafkaFlow.UnitTests/KafkaFlow.UnitTests.csproj [hunk 2]
```diff
@@ -15,17 +15,19 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="AutoFixture" Version="4.18.0" />
-        <PackageReference Include="AutoFixture.AutoMoq" Version="4.18.0" />
+        <PackageReference Include="AutoFixture" Version="4.18.1" />
+        <PackageReference Include="AutoFixture.AutoMoq" Version="4.18.1" />
+		    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```
Comment (joelfoliveira) [inline tests/KafkaFlow.UnitTests/KafkaFlow.UnitTests.csproj]: please fix indentation

**Review by joelfoliveira — APPROVED** [review]

**Review by JoaoRodriguesGithub — APPROVED** [review]

File: tests/KafkaFlow.UnitTests/KafkaFlow.UnitTests.csproj (hunk 2 above)
Comment (erik-catalao) [reply tests/KafkaFlow.UnitTests/KafkaFlow.UnitTests.csproj]: Fixed, ty for the warning!

File: tests/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj (hunk 1 above)
Comment (erik-catalao) [reply tests/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj]: Fixed, ty for the warning!

**Pull request merged**